docs/docs/concepts/password-policy.mdx

```diff
@@ -0,0 +1,49 @@
+---
+id: password-policy
+title: Password Policy
+---
+
+Password-based authentication flows are subject to frequent abuse through social
+engineering, password guessing and phishing attacks.  
+Ory Cloud implements measures to provide high security for password-based
+flows.  
+The Ory Cloud password policy is based on recommendations by the
+[`NIST`](https://pages.nist.gov/800-63-3/sp800-63b.html) and
+[`NCSC`](https://www.ncsc.gov.uk/collection/passwords/updating-your-approach)
+institutes among
+[others](https://www.troyhunt.com/passwords-evolved-authentication-guidance-for-the-modern-era/).
+
+- The password must at least be 8 characters long and all characters (unicode,
+  ASCII) must be allowed.
+- Ory Cloud makes sure the password is not similar to the username/email or
+  other credentials.  
+  To ensure the password is different, Ory Cloud enforces a minimum
+  [Levenshtein distance](https://en.wikipedia.org/wiki/Levenshtein_distance)
+  (measuring how much two strings resemble each other). It also makes sure no
+  significant strings of the credentials are part of the password.
+- Ory Cloud checks all passwords against the
+  [`HIBP`](https://haveibeenpwned.com/) API.  
+  `HIBP` provides a open database of leaked passwords. Every password sent to
+  `HIBP` is SHA-1 hashed and anonymized, so there is no compromise of user
```

Comment:
I suggest to focus on the feature rather than how we achieve it. So something like: "Ory checks all passwords in an anonymized way against a database of known leaked passwords using the Have I Been Pwned API.